lib/firebase_options.dart

```diff
@@ -0,0 +1,64 @@
+// File generated by FlutterFire CLI.
+// ignore_for_file: lines_longer_than_80_chars, avoid_classes_with_only_static_members
+import 'package:firebase_core/firebase_core.dart' show FirebaseOptions;
```

Comment:
Do we need to have this file now? iirc, pinbal had the firebase configuration done directly via JS files.

Reply:
I haven't been able to run the project without it after upgrading firebase_core